src/test/java/org/gridsuite/modification/server/utils/MatcherTwoWindingsTransformerCreationInfos.java

```diff
@@ -29,6 +31,16 @@ public boolean matchesCurrentLimitsInfos(CurrentLimitsInfos cl1, CurrentLimitsIn
                 || (cl1 != null && cl2 != null && cl1.getPermanentLimit() == null && cl2.getPermanentLimit() == null);
     }
 
+    public boolean matchesRatioTapChangerInfos(RatioTapChangerInfos rt1, RatioTapChangerInfos rt2) {
```

Comment:
minor remark : We could have this in a separated `MatcherRatioTapChangerCreationInfos` class because tomorrow, we will need to share this code with 3WT

Reply:
I'll include it in the technical story

Reply:
deal